extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/schema/statistics.class.property.schema.json

```diff
@@ -61,7 +82,10 @@
         "type": [
           "number",
           "array"
-        ]
+        ],
+        "items": {
+          "type": "number"
+        }
```

Comment:
The type should "integer" in both places here. I just pushed a fix related to that https://github.com/CesiumGS/glTF/commit/b3f48c2a594ac1c6319dd0087b4f5fef2218a832